### PR TITLE
Add proper handling for common and pod specific labels and annotations.

### LINKED ISSUE #259: Ability to label deployed resources
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): feature

Many other helm charts have the ability to label deployed resources, which we use for our Velero backup system, for example list this:
```sh
    commonLabels:
      stackspin.net/backupSet: "zammad"
    podLabels:
      stackspin.net/backupSet: "zammad"
```
zammad does not seem to honor that, would be great if it did

### PR DESCRIPTION
**Pull request by mgruner**
#### Which issue this PR fixes

- fixes #259

#### Checklist

- [x] Chart Version bumped
- [x] Clarify if this satisfies the requirements from #259

### CODE REVIEW

File: zammad/values.yaml [hunk 1]
```diff
@@ -402,6 +402,12 @@ autoWizard:
   #     ]
   #   }
 
+commonLabels: {}
+  # my-label: "value"
+commonAnnotations: {}
+  # my-annotation: "value"
+podLabels: {}
```
Comment (monotek) [inline zammad/values.yaml:409]: imho thats not very flexible as you only can set same labels on all pods.
should be available per pod.
in the end would be same as commonLabels, or not?

File: zammad/values.yaml [hunk 2]
```diff
@@ -402,6 +402,12 @@ autoWizard:
...
+  # my-label: "value"
 podAnnotations: {}
```
Comment (monotek) [inline zammad/values.yaml:411]: imho thats not very flexible as you only can set same annotations on all pods.
should be available per pod.
in the end would be same as commonAnnotations, or not?

**Review by monotek — CHANGES_REQUESTED** [review]

File: zammad/values.yaml (hunk 1 above)
Comment (mgruner) [reply zammad/values.yaml]: Should be fixed by a36b0b94cd0ec3a25119f61f54c47ae37360a6c7, can you check please?

File: zammad/values.yaml (hunk 2 above)
Comment (mgruner) [reply zammad/values.yaml]: Should be fixed by a36b0b94cd0ec3a25119f61f54c47ae37360a6c7, can you check please?

@monotek Could you please add my user mgruner as an admin to the zammad organisation on artifacthub?

File: zammad/values.yaml (hunk 1 above)
Comment (mgruner) [reply zammad/values.yaml:429]: @monotek the final state should allow for anything an admin can want to have regarding labels and annotations, global, global for pods, and pod specific.
Please let me know if this looks ok.

**Review by monotek — APPROVED** [review]

File: zammad/values.yaml (hunk 1 above)
Comment (monotek) [reply zammad/values.yaml:429]: Nice :)

**Pull request merged**